### DISCUSSION TITLE
'so-test' code

### DISCUSSION THREAD

**Discussion by leber2** [Security-Onion-Solutions/securityonion#8542] [Unsupported Versions] [answered]
Was wondering what is the code behind 'so-test' command. Is there any way looking for it? Thanks. :) 

Any help would be very much appreciated. Thanks. :)

**Comment by reyesj2** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/common/tools/sbin/so-test

https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/common/tools/sbin/so-tcpreplay
